app/controllers/lessons_controller.rb

```diff
@@ -47,42 +47,46 @@ def new
   end
 
   def create
-    lesson = Lesson.new
-    lesson.hash_tag = params[:lesson][:hash_tag]
-    lesson.created_at = Time.now
-    lesson.category = params[:lesson][:category]
-    lesson.description = params[:lesson][:description]
-    lesson.approved = true
-    lesson.save
-    cookies["new_lesson_id"] = lesson.id
+    if user_signed_in?
```

Comment:
Hi, I'm looking through your review during learn day (monthly day where we get free time to work on our own projects at my office) and I agree with the refactors you suggested.  I opened issue #73 and assigned myself to it, but I will work on this PR more before working on issue #73.  Thanks again!